src/Meshes/Builders/capsuleBuilder.ts

```diff
@@ -131,7 +132,7 @@ export function CreateCapsuleVertexData(
             normal.set(sinTheta, slope, cosTheta).normalize();
             normals.push(normal.x, normal.y, normal.z);
             // uv
-            uvs.push(u, 1 - v / vl);
+            uvs.push(u, v / vl ? v / vl : 1 - v / vl);

```

Comment:
Should be `CompatibilityOptions.UseOpenGLOrientationForUV ? v / vl : 1 - v / vl`

Reply:
lol yes!! bad copy paste